### PR TITLE
(SIMP-8416) Update to remove packagecloud

### PR DESCRIPTION
**Pull request by trevor-vaughan**
* Fixed:
  * Migrated to the new simp-beaker-helpers method instead of the old
    internal method for the SIMP repos.
  * Updated the test RPMs to use the latest Gemfile and Rakefile
  * Rebuilt the test RPMs on EL7
* Notes:
  * The acceptance tests will fail if running on EL8/Fedora32+ when
    testing EL7 due to an incompatibility in RPM.

SIMP-8416 #comment updated simp-adapter

### CODE REVIEW

File: spec/fixtures/test_module_rpms/pupmod-simp-beakertest-0.0.1/Gemfile.lock [hunk 1]
```diff
@@ -1,463 +1,270 @@
-GIT
-  remote: https://github.com/trevor-vaughan/rubygem-simp-rake-helpers
-  revision: 34e01602fe294d294e74caefd060960e23bed0c5
```
Comment (lnemsick-simp) [inline spec/fixtures/test_module_rpms/pupmod-simp-beakertest-0.0.1/Gemfile.lock:3]: This file should not be removed.  It is a historical record of the (buggy) version of simp-rake-helper used to create this RPM.

**Review by lnemsick-simp — CHANGES_REQUESTED** [review]
Please do not mess with the pupmod-simp-beakertest-0.0.1 files at all. Per the README in spec/fixtures/test_module_rpms, that RPM has the simp_rpm_helper in the %post.  In other words, it was built using a buggy version of simp-rake-helpers.  It is required to test RPM upgrades from Puppet module RPMs with the bug to Puppet module RPMs without the bug.

**Comment by trevor-vaughan** [comment]
@lnemsick-simp Ah, sorry about that.

**Comment by lnemsick-simp** [comment]
Some of the upgrade tests (from 0.6.0 version to current) are failing because the old versions of Puppet RPMs required are no longer in the official puppet repos.  Also, the logic tested is no longer needed.

Will remove the logic and the corresponding tests.  Marking as WIP until the work is completed.

**Comment by lnemsick-simp** [comment]
Now that the upgrade tests from 0.6.0 have been removed, this can be updated.  Won't need all the changes made.  I will fix.

**Comment by lnemsick-simp** [comment]
https://gitlab.com/simp/simp-adapter/-/pipelines/209276453

**Review by lnemsick-simp — APPROVED** [review]

**Pull request merged**